k8s/data-pipeline/deployments/oauth2-proxy.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: oauth2-proxy
```

Comment:
Does this label need to be here? I believe all the label selector stuff for the deployment is already handled under `spec`.

Reply:
I think you're right.